db-con.php

```diff
@@ -0,0 +1,9 @@
+<?php
+
+$conn = new PDO('mysql:host=localhost;dbname=db_name', 'root', '');
```

Comment:
Can you think of a name for the DB that describes the purpose and content of the database? db_name does not really give a lot of information about what we have inside and what we use it for currently.